packages/adapter/addon/rest.js

```diff
@@ -1067,9 +1062,19 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
   },
 
   _ajax(options) {
-    if (get(this, 'useFetch')) {
+    if (this.useFetch) {
       this._fetchRequest(options);
-    } else if (get(this, 'fastboot.isFastBoot')) {
+    } else if (DEPRECATE_NAJAX && get(this, 'fastboot.isFastBoot')) {
+      if (has('fetch')) {
+        deprecate(
+          'You have ember-fetch and jquery installed. To use ember-fetch, set `useFetch: true` in your adapter.  In 4.0, ember-data will fallback to ember-fetch instead of najax when both of these are installed.',
+          false,
+          {
+            id: 'ember-data:najax-fallback',
+            until: '4.0',
+          }
+        );
+      }
```

Comment:
we should add and `else` that straigh up deprecates `najax` even when fetch is not present and tells folks to use something like ember-fetch